lib/middlewares/object_authorization.ex

```diff
@@ -114,6 +122,13 @@ defmodule Rajska.ObjectAuthorization do
     authorize(schema_node, selections ++ tail, resolution)
   end
 
+  defp find_associations(
+    [%Absinthe.Blueprint.Document.Fragment.Spread{} | tail],
```

Comment:
I'll merge it for now since it is covering our use cases, but we can continue that discussion in another PR if you still have troubles with it. 

I'm not sure how to deal with interface types, it looks like we would need to rely only on the ObjectScopeAuthorization for this case, since it is the only one that runs after resolution